js.js

```diff
@@ -0,0 +1,18 @@
+const body = document.querySelector('body');
+const span = document.querySelector('span');
+const storage = window.sessionStorage;
+
+body.addEventListener('click', clicked);
+
+let counter = storage.getItem('counter') || 0;
+addCounter(span);
+
+function clicked() {
+  counter ++;
+  addCounter(span);
```

Comment:
In this task, we don't have any other elements to display the number. Probably the function shouldn't accept any arguments. Let's get rid of `span` and pass the value directly in `body`?

Reply:
done